contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,96 @@
+
+So what about stepping on toes? Well, we [actively encourage that](https://posthog.com/handbook/values). Everyone at PostHog should be raising pull requests in GitHub or proposing changes to stuff that doesn't have anything to do with your small team. Why? Because it shows they really care and they really understand the product.
+
+
+### They have their own mission
+
+Regardless of size or scope, each team has its own mission that feeds into our overall company mission of equipping every developer to build successful products.
+
+On the [marketing team](https://posthog.com/teams/marketing), the mission is to make PostHog a place for product engineers to learn and collaborate. The [infrastructure team’s mission](https://posthog.com/teams/infrastructure) is to make deploying, scaling, and managing PostHog easy, fast, and reliable. But a charter is more than a mission. Small team charters also need to include:
+
+* Long-term goals
+* A description of what the team does
+* A target customer
+* Details of who’s on the team
+* Key metrics
+
+All of this information is in our [handbook](https://posthog.com/handbook) and updated when changes are made and confirmed each quarter. This keeps everyone on the same page and makes it easy for anyone within the company to quickly find out what other teams are working on and why.
+
+### They’re flexible
+
+We’d rather hire new people than keep moving people around to fill gaps. That said, we’re happy for people to move between teams when needed, ideally no more often than every three to nine months. There are two scenarios that could trigger a move:
+
+* A small team may realize they no longer need someone, or that they could use the help of someone internally on another small team.
+* Someone might want to move to another small team to develop their skills or experience.
+
+Rather than the team lead making the call, it’s always the manager who decides if one of their direct reports should move between teams. But, once in a small team, you’re only in one team. Someone being in multiple small teams at once defeats the purpose of ownership.
+
+If someone feels the need to be in more than one small team, that likely means we need to hire. That can mean we’re ‘overstaffed’ at times, but ultimately good people will do good work to push our product forward. And it’s better to have slightly overstaffed small teams than to be perpetually understaffed and for your product to suffer as a result.
+
+Small teams are also able to work with other small teams on multidisciplinary projects and members of one small team can and should attend meetings on other small teams to help with collaboration if needed.
```

Comment:
```suggestion
Small teams are also able to work with other small teams on multidisciplinary projects and members of one small team can and should attend meetings on other small teams to help with collaboration if needed.

### 

### The tradeoffs of small teams

Our small teams structure is a success because we work hard on making it one. There are tradeoffs. 

Working as a series of startups within a startup risks creating silos and there could be whole teams or individual team members who drift away from our mission or feel disconnected from the company. 

To help alleviate this, all of our small teams have annual off-sites in addition to our big company get-together. Everyone also has a generous quarterly travel budget and people are encouraged to go work with colleagues either on the same small team as them or in another part of the company. We mix things up and spend time together IRL to work on problems and get things shipped.

Small teams also require that everyone is very good at written communication, be it a Slack message, or a Github pull request or issue. This helps everyone at the company to both see and understand what's being worked on and why.

You also need to hire the right people to work on small teams. Not everyone will embrace this way of working. It requires people who take extreme ownership of ideas, are self-starting, authentic, and low-ego. We call them cracked engineers. And we have some [strong thoughts](https://newsletter.posthog.com/p/hiring-and-managing-cracked-engineers) on what makes an engineer cracked.
```

Reply:
Don't think this works atm. I think:

- Let's bullet point this with bold headings, so it's easier to scan.
- These tradeoffs feel a little surface level atm. I think it's worth chatting to James and/or Tim to get their takes / brainstorm some more fundamental things.
- Also, there are probably one or two problems with small teams that we just accept in return for the benefits.

Reply:
@jamesefhawkins tagging you here for any immediate thoughts, but also happy to jump on a call/take to Slack DMs and talk through the tradeoffs of small teams, what the pitfalls might be, what we do to mitigate against them, why small teams won't work for every startup, etc. and when this structure might stop working for us.